lib/handlers/lnurl_handler.dart

```diff
@@ -50,7 +50,7 @@ class LNURLHandler {
       });
 
       Navigator.of(context).push(FadeInRoute(
-        builder: (_) => CreateInvoicePage(lnurlWithdraw: response),
+        builder: (_) => PayInvoicePage(lnurlPay: response),
```

Comment:
> Why did you change the withdraw handling? The PayFetchResponse doesn't expect `WithdrawFetchResponse` as argument and it will throw an error.

That's a unintentional error. Sorry. Reverted.